v1.6.0 gem missing files

When I run 1.6.0, I get this error:

```
node:internal/modules/cjs/loader:927
  throw err;
  ^

Error: Cannot find module './package.json'
Require stack:
- /usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/bin-prettier.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/bin-prettier.js:24:19)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/bin-prettier.js'
  ]
```

Key bit: `/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/bin-prettier.js` is trying to load `./package.json`.

I'm not 100% confident on how relative requires work, but I think that means it wants `/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/package.json`.

Which is not there:

```console
% gem contents prettier
<snip>
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/bin-prettier.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/index.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-angular.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-babel.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-espree.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-flow.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-glimmer.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-graphql.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-html.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-markdown.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-meriyah.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-postcss.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-typescript.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/parser-yaml.js
/usr/local/bundle/gems/prettier-1.6.0/node_modules/prettier/third-party.js
<snip>
```

If I understand the gemspec, you intend to build the `node_modules` stuff locally then include them:

```rb
  spec.files =
    Dir.chdir(__dir__) do
      %w[LICENSE bin/console package.json rubocop.yml] +
        Dir['{{exe,lib,src}/**/*,*.md}'] +
        Dir[
          'node_modules/prettier/{index,bin-prettier,third-party,parser-*}.js'
        ]
    end
```

And this doesn't appear to be including everything it needs.

I've managed to hack a random `package.json` into the tree, but I have no idea if it would get things functional because I just run into another missing file (`doc.js`).

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -27,7 +27,7 @@ jobs:
         with:
           bundler-cache: true
           ruby-version: ${{ matrix.ruby }}
-      - uses: actions/setup-node@v2-beta
+      - uses: actions/setup-node@v2
         with:
           node-version: 12.x
       - id: yarn-cache
@@ -54,7 +54,7 @@ jobs:
         with:
           bundler-cache: true
           ruby-version: "3.0"
-      - uses: actions/setup-node@v2-beta
+      - uses: actions/setup-node@v2
         with:
           node-version: 12.x
       - id: yarn-cache
@@ -68,6 +68,31 @@ jobs:
       - run: |
           yarn check-format
           yarn lint
+
+  gem:
+    name: Gem
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: "3.0"
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+      - id: yarn-cache
+        run: echo "::set-output name=directory::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.directory }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+      - run: yarn install --frozen-lockfile
+      - run: |
+          gem build -o prettier.gem
+          gem unpack prettier.gem
+          prettier/exe/rbprettier --help
 
   # Saves pull request details for later workflow runs which can run in
   # a privileged environment

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 - [#862](https://github.com/prettier/plugin-ruby/issues/862) - azz, kddeisz - Group together `.where.not` calls in method chains.
 - [#863](https://github.com/prettier/plugin-ruby/issues/863) - azz, kddeisz - Fix up Sorbet `sig` block formatting when chaining method calls.
 - [#908](https://github.com/prettier/plugin-ruby/issues/908) - Hansenq, kddeisz - Method chains with blocks should be properly indented.
+- [#916](https://github.com/prettier/plugin-ruby/issues/916) - pbrisbin, kddeisz - Ensure all of the necessary files are present in the gem.
 
 ## [1.6.0] - 2021-06-23
 

--- a/prettier.gemspec
+++ b/prettier.gemspec
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
       %w[LICENSE bin/console package.json rubocop.yml] +
         Dir['{{exe,lib,src}/**/*,*.md}'] +
         Dir[
-          'node_modules/prettier/{index,bin-prettier,third-party,parser-*}.js'
+          'node_modules/prettier/{package.json,index.js,doc.js,bin-prettier.js,third-party.js,parser-*.js}'
         ]
     end
 

